scene/2d/physics/physical_bone_2d.cpp

```diff
@@ -107,7 +107,7 @@ void PhysicalBone2D::_find_joint_child() {
 }
 
 PackedStringArray PhysicalBone2D::get_configuration_warnings() const {
-	PackedStringArray warnings = Node::get_configuration_warnings();
+	PackedStringArray warnings = Node2D::get_configuration_warnings();
```

Comment:
Also this is wrong, though that will be fixed by just rebasing and removing all these changes